versioned/storage/dynamodb/build.gradle.kts

```diff
@@ -44,11 +44,8 @@ dependencies {
   implementation(libs.slf4j.api)
 
   implementation(platform(libs.awssdk.bom))
-  implementation("software.amazon.awssdk:dynamodb") {
-    exclude("software.amazon.awssdk", "apache-client")
-  }
-  implementation("software.amazon.awssdk:netty-nio-client")
-  implementation("software.amazon.awssdk:url-connection-client")
```

Comment:
we seem to be using `software.amazon.awssdk:url-connection-client` in lots of other places as well... wouldnt it make sense to switch all awssdk usage to the same apache http client?

Reply:
Others are testing modules

Reply:
if the apache client is the preferred choice for production modules, why not also for testing modules?